src/_variables.scss

```diff
@@ -582,3 +584,9 @@ $snackbar-action-color: unquote("rgb(#{$color-accent})") !default;
 /* TOOLTIP */
 $tooltip-font-size: 10px !default;
 $tooltip-font-size-large: 14px !default;
+
+/* CHIP */
+$chip-bg-color: rgb(222, 222, 222) !default;
+$chip-bg-active-color: rgb(214, 214, 214) !default;
+$chip-height: 32px !default;
+$chip-font-size: 13px !default; 
```

Comment:
Let's go ahead and make this `0.928rem` to align with the spec. `sp` values are the equivalent of `rem` and our base unit is currently 14px for calculations.